base/environment.yml

```diff
@@ -1,6 +1,10 @@
 channels:
 - conda-forge
 dependencies:
+- python =3.9
 - ipywidgets
-- pyiron_base =0.5.7
-- seaborn =0.11.2
+- pyiron_base =0.5.32
+- sqlalchemy <2.0.0
+- seaborn =0.12.2
+- damask
+- openssh
```

Comment:
The `pyiron_base` environment should be minimal. So it should not contain specific tools like `damask`